<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>731</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Proxy Config</string>
    </property>
    <widget class="QPushButton" name="btnConfig">
     <property name="geometry">
      <rect>
       <x>624</x>
       <y>37</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gboxProtocol">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Protocol</string>
     </property>
     <widget class="QRadioButton" name="rdoTcp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>69</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoUdp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>43</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoSsl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>69</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>SSL</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gboxListen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>19</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Listen on</string>
     </property>
     <widget class="QLineEdit" name="txtListenAddress">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtListenPort">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5039</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>37</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gboxSendTo">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Send to</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>37</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSendAddress">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSendPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5040</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="chkPauseSendToRemote">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>70</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause &quot;to remote&quot;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkPauseSendToLocal">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>90</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause &quot;to local&quot;</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="txtClient">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>160</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtServer">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>145</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>147</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtToRemote">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendToRemote">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtToLocal">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendToLocal">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>731</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QTextEdit" name="txtLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>711</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
